### PR TITLE
py: Fix automatic addition of __doc__ to dict object (#229)

### LINKED ISSUE #229: Error when i try to get the `__doc__` from a type like `str`
```bash
Python 3.4.0 (, )
[Gpython ]
- os/arch: linux/amd64
- go version: go1.21.5
>>> "".__doc__
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: "'str' has no attribute '__doc__'"
```

### PR DESCRIPTION
**Pull request by natanfeitosa**
fixes #229

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -23,6 +23,9 @@ import (
 
 var (
 	cpuprofile = flag.String("cpuprofile", "", "Write cpu profile to file")
+	version = ""
+	commit = ""
+	date = ""
```
Comment (sbinet) [inline main.go:28]: please remove (they are generated elsewhere)

**Review by sbinet — COMMENTED** [review]
thanks for the PR.

could you address the comments below ?

thanks.

**Review by natanfeitosa — COMMENTED** [review]
Removed unnecessary code

**Comment by sbinet** [comment]
thanks a lot.

could you send another PR against https://github.com/go-python/license adding yourself to the `AUTHORS` and/or `CONTRIBUTORS` file(s) ?
(then I'll merge this one in)

**Comment by sbinet** [comment]
thanks

**Pull request merged**